docs/src/main/asciidoc/spring-integration-pubsub.adoc

```diff
@@ -108,6 +111,74 @@ public MessageHandler messageReceiver() {
 }
 ----
 
+====== Nacking
+In the context of Cloud Pub/Sub, nacking simply means instructing the message to be redelivered immediately.
+This could be useful in a load balanced architecture, where one of the subscribers is having issues but others are available to process messages.
+
+In `AUTO` mode, a message is nacked if processing causes any unchecked exceptions.
+
+In `AUTO_ACK` and `MANUAL` modes, behavior depends on whether an error channel was configured on the `PubSubInboundChannelAdapter`:
+
+* If an error channel is present (see <<#Error-Handling>> for how to configure one), the message is neither acked nor nacked.
```

Comment:
Can all of these complicated scenarios be captured in a table where on one axis is Mode and another is Error/No-Error/Handled/etc...? And the value is whether it's nacked/acked/neither.